main.py

```diff
@@ -99,7 +109,11 @@ async def restore_config():
 
 @app.get("/logs")
 async def log_endpoint(force_latest: bool):
-    return Log(log=get_last_log(force_latest), latency=app.state.initializer.runner.latency)
+    if app.state.initializer.runner:
+        latency = app.state.initializer.runner.latency
+    else:
+        latency = 0
```

Comment:
```suggestion
    try:
        latency = app.state.initializer.runner.latency
    except XYZError:
        latency = 0
```
No need to access `app.state.initializer.runner` twice.